Assignment/Conversion.c

```diff
@@ -0,0 +1,107 @@
+// ------------------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// ------------------------------------------------------------------
+// Conversion.c
+// Program for Number Conversion.
+// ------------------------------------------------------------------------------------------------
+
+#pragma warning(disable:4996)
+#include <stdio.h>
+
+void DecToBinary (int n);
+void DecimalToBinary (int n);
+void DecToHexa (int n);
+int mBitLength;
+
+/// <summary>Decimal number to binary Conversion</summary>
+void DecToBinary (int n) {
+   int binary[32], i = 0, isNegative = 0; // Flag is to check if the number is negative
+   if (n < 0) {
+      n = -n;
+      isNegative = 1;
+   }
+   while (n > 0) {
+      binary[i++] = n % 2;
+      n /= 2;
+   }
+   // If the number was negative,apply two's complement
+   if (isNegative) {
+      if (i <= 7) mBitLength = 8;

```

Comment:
You can use Ternary operator here, like so:
```suggestion
      mBitLength = i <= 7 ? 8 : 16;
```